package.json

```diff
@@ -49,8 +49,7 @@
     "blockstore-core": "^1.0.5",
     "ipfs-unixfs": "^11.0.0",
     "miniswap": "^2.0.0",
-    "standard": "^17.0.0",
-    "uint8arrays": "^3.0.0"
```

Comment:
turns out the swiss-army-knife has `toString` and `fromString` under `multiformats/bytes`! who knew!